app/src/ui/index.tsx

```diff
@@ -99,15 +103,94 @@ process.once('uncaughtException', (error: Error) => {
       `An uncaught exception was thrown. If this were a production build it would be reported to Central. Instead, maybe give it a lil lookyloo.`
     )
   } else {
-    sendErrorReport(error, {
+    const extra: Record<string, string> = {
       osVersion: getOS(),
       guid: getGUID(),
-    })
+    }
+
+    try {
+      if (currentState) {
+        if (currentState.currentBanner !== null) {
+          extra.currentBanner = currentState.currentBanner.type
```

Comment:
Won't these `type` properties return the enum number and not the human readable value?

Reply:
The currentBanner enum type is a string but the other are numbers. We'll have to do some work to map it ourselves. We can talk about mapping these to human readable strings in the future

Reply:
Yea, I only through it on that line since it was the first occurrence. I assumed most of our enums would be a number instead of a human-readable string.

Also, this is very much a non-blocker for me.